js/source/worker_tile.js

```diff
@@ -108,9 +108,9 @@ WorkerTile.prototype.parse = function(data, layerFamilies, actor, rawTileData, c
         }
     }
 
-    var buckets = [],
+    var buckets = this.buckets = [],
```

Comment:
> investigate whether the arraybuffers continue to consume memory after they're transferred/neutered

Ah! I didn't think about this. The memory impact should be minimal thanks to the way transferables work. #2432